indico/modules/events/roles/templates/roles.html

```diff
@@ -0,0 +1,103 @@
+{% extends 'events/management/base.html' %}
+{% from 'events/roles_roles.html' import render_roles %}
+
+{% block title %}{% trans %}Roles{% endtrans %}{% endblock %}
+
+{% block content %}
+    {# TODO: move to scss file #}
+    <style>
+        .i-tag:not(:first-child) {
+            margin-left: 0.2em;
+        }
+        .role-list-row {
+            background: #eee;
+        }
+        .role-list-row td {
+            padding: 0 !important;
+        }
+        .role-list-row td .content-area {
+            background: #fff;
+            margin: 0 0 0 58px !important;
+        }
+        .role-list {
+            padding: 10px;
+            box-sizing: border-box;
+            list-style-type: none;
+        }
+        .role-list li {
+            display: inline-block;
+            margin-right: 0.3em;
+        }
+        .role-list li:before {
+            margin-right: 0.3em;
+        }
+        .badge-column {
+            width: 20px;
+        }
+        .role-badge {
+            display: inline-block;
+            width: 25px;
+            text-align: center;
+        }
+        .role-members-summary {
+            text-align: right;
+        }
+        .toggle-members {
+            cursor: pointer;
+        }
+    </style>
+
+    <table class="i-table-widget multi-tbody" id="event-roles">
+        {{ render_roles(roles) }}
+    </table>
+    <div class="toolbar f-j-end space-before">
+        <button class="i-button highlight icon-plus hide-if-locked"
+                data-href="{{ url_for('.add_role', event) }}"
+                data-title="{% trans %}Add a new role{% endtrans %}"
+                data-update="#event-roles"
+                data-ajax-dialog>
+            {% trans %}Add Role{% endtrans %}
+        </button>
+    </div>
+
+    {# TODO: move to JS file #}
```

Comment:
I guess this is the `roles.js` file that @meluru is creating in her branch?

Reply:
Probably - initially we didn't add the separate CSS/JS file to avoid rebase hell in bundles.py with the changes from my config refactoring branch.